src/game/boe.graphics.cpp

```diff
@@ -709,6 +709,15 @@ void draw_terrain(short	mode) {
 	
 	mainPtr.setActive();
 	
+	int max_dim_x, max_dim_y;
+	if(is_out()){
+		// TODO I don't know for sure whether width() and height() need to be flipped here!
```

Comment:
If it helps, x is "column" and y is "row".

Reply:
if x and y are intuitively mapped for the array of outdoor sections then I'm pretty sure this is correct.

Reply:
I verified it